apps/mark-scan/frontend/src/electrical_testing/app_root.tsx

```diff
@@ -0,0 +1,27 @@
+import { Main } from '@votingworks/ui';
+
+import { getElectricalTestingStatusMessages } from './api';
+
+export function AppRoot(): JSX.Element {
+  const getElectricalTestingStatusMessagesQuery =
+    getElectricalTestingStatusMessages.useQuery();
+
+  return (
+    <Main centerChild style={{ height: '100%', padding: '1rem' }}>
+      <img src="../../src/electrical_testing/mario.gif" alt="Mario" />
+      <br />
+      <ul style={{ listStyleType: 'none', margin: 0, padding: 0 }}>
+        {(getElectricalTestingStatusMessagesQuery.data ?? []).map(
+          ({ component, statusMessage, updatedAt }) => (
+            <li key={component}>
+              [{updatedAt}] {component}: {statusMessage}
+            </li>
+          )
+        )}
+      </ul>
+
+      {/* eslint-disable-next-line jsx-a11y/media-has-caption */}
+      <audio autoPlay loop src="/sounds/alarm.mp3" />
+    </Main>
```

Comment:
I know there's some logic to share here across VxMark and VxScan. Waiting for the dust to settle a bit before I abstract and share code more fully here. Same applies to the store methods on the backend